pkg/repository/repository.go

```diff
@@ -0,0 +1,14 @@
+package repository
+
+import "database/sql"
+
+type Repository struct {
```

Comment:
Czy to `repository` to nazwa generyczna, która powinna być zastąpiona czymś bardziej konkretnym, jak `user_repository` or `user`?